odc/geo/geobox.py

```diff
@@ -64,12 +64,40 @@
     xy_,
 )
 
-GeoboxAnchor = Union[AnchorEnum, XY[float]]
+GeoboxAnchor = Union[
+    AnchorEnum,
+    XY[float],
+    float,
+    Literal["center"],
```

Comment:
This is fantastic - will make "anchor" even more usable than the current "align". Is the double "centre" here intended?

Reply:
yes, both are "valid" spellings

Reply:
Ah missed the spelling difference, makes sense